lisa/sut_orchestrator/azure/arm_template.json

```diff
@@ -70,6 +70,34 @@
                 "description": "tags of availability set"
             }
         },
+        "use_existing_virtual_network": {
+            "type": "bool",
+            "defaultValue": false,
+            "metadata": {
+                "description": "flag to use an existing vnet"
+            }
+        },
+        "virtual_network_resource_group": {
+            "type": "string",
+            "defaultValue": "[resourceGroup().name]",
+            "metadata": {
+                "description": "the name of vnet resource group"
+            }
+        },
+        "virtual_network_name": {
+            "type": "string",
+            "defaultValue": "lisa-virtualNetwork",
```

Comment:
not sure the default values serve any real purpose (since the defaults seemed to only work when they came from the py code)